src/Utility/RequestLetter.js

```diff
@@ -173,30 +174,33 @@ export default class RequestLetter {
         let name = '';
 
         request_data.forEach(item => {
```

Comment:
Now that I look at it again I feel like this could have been done a little more elegantly using `Array.prototype.reduce()`. Oh, well.

Reply:
Yeah, I also have quite a few places where I would much rather use a `.map()` or `.reduce()` instead of an ugly `forEach`. We'll get there eventually. :D